### PR TITLE
Reducing createAsset() repetition

### PR DESCRIPTION
**Pull request by CalebBredekamp**

### CODE REVIEW

File: Service/pom.xml [hunk 1]
```diff
@@ -217,7 +217,7 @@
         <dependency>
             <groupId>org.testcontainers</groupId>
             <artifactId>testcontainers</artifactId>
-            <version>1.11.4</version>
+            <version>1.15.2</version>
             <scope>test</scope>
```
Comment (CalebBredekamp) [inline Service/pom.xml:221]: testcontainers changed the location of the ryuk image as noted [here](https://stackoverflow.com/questions/61887363/testcontainers-cant-pull-ryuk-image-quay-io-is-not-reachable) I've used the latest realease for now.

File: Service/src/test/java/za/co/imqs/api/asset/AbstractAssetControllerAPITest.java [hunk 2]
```diff
@@ -318,6 +318,37 @@ public CoreAssetBuilder dept(String dept) {
         }
     }
```
Comment (CalebBredekamp) [inline Service/src/test/java/za/co/imqs/api/asset/AbstractAssetControllerAPITest.java:320]: I couldn't find a nice way to fully generalise these methods so I just made them as specific as possible to not muck around in the middle :) This does mean the AbstractAssetController now has methods coupled to the AssetEnvelopeDto though. Not too sure if that is a bad idea

File: Service/src/test/java/za/co/imqs/api/asset/AssetControllerQuantityCrudAPITest.java [hunk 3]
```diff
@@ -33,22 +33,9 @@ public void clearAsset() {
...
-                get();
-
-        given().
-                header("Cookie", login.getSession()).
-                contentType(ContentType.JSON).body(envelope).
-                put("/assets/{uuid}", THE_ASSET).
-                then().assertThat().
-                statusCode(HttpStatus.SC_CREATED);
-
+        //TODO: Do I need the serial here? Why was it added before?
```
Comment (frankvanrietimqs) [inline Service/src/test/java/za/co/imqs/api/asset/AssetControllerQuantityCrudAPITest.java:36]: Serial is not rquired here - it was probaly used in a test of update asset and got copied here

File: Service/src/test/java/za/co/imqs/api/asset/AbstractAssetControllerAPITest.java (hunk 2 above)
Comment (frankvanrietimqs) [reply Service/src/test/java/za/co/imqs/api/asset/AbstractAssetControllerAPITest.java:320]: I agree with your approach. Keep is simple and explicit

**Review by frankvanrietimqs — APPROVED** [review]

**Pull request merged**